_includes/list-partners.html

```diff
@@ -0,0 +1,19 @@
+{% if include.style == "list" %}
+  <ul>
+    {% for agency in project.partner %}
+      {% capture agencyname %}{{ agency | slugify }}{% endcapture %}
+      {% capture partnerurl %}{% for partner in site.data.partners %}{% if partner[0] == agencyname %}{{ partner[1].url }}{% else %} {% endif %}{% endfor %}{% endcapture %}
```

Comment:
I'd love to but couldn't figure out a way to get `agency` without being inside the `{% for agency in project.partner %}`